src/components/dashboard/StatusPanel.js

```diff
@@ -0,0 +1,64 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+
+import { withStatusInfo } from '../hoc/status';
+
+import * as websocket from '../../services/websocket';
+import * as statusService from '../../services/status';
+
+import Panel from '../commons/Panel';
+import ServiceList from './ServiceList';
+import Spinner from '../commons/Spinner';
+
+class StatusPanel extends Component {
+  componentDidMount() {
```

Comment:
I would prefer not to use withState as I don't like the way of how setting state works. And its just my opinion. In the other side, it makes code cleaner and its the current trend. So, I think we can use it.